### PR TITLE
dev-db/ocp: bump to 0.3

### PR DESCRIPTION
**Pull request by maxsatula**
Replaced a dependency on a transitional package `dev-db/oracle-instantclient-basic` with a newer `dev-db/oracle-instantclient`

Signed-off-by: Max Satula <maksym.satula@gmail.com>

### CODE REVIEW

File: dev-db/ocp/ocp-0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (a17r) [inline dev-db/ocp/ocp-0.3.ebuild]: How about EAPI 7?

File: dev-db/ocp/ocp-9999.ebuild [hunk 2]
```diff
@@ -20,10 +20,12 @@ SLOT="0"
 IUSE=""
 
 RDEPEND="
-	dev-db/oracle-instantclient-basic
```
Comment (a17r) [inline dev-db/ocp/ocp-9999.ebuild]: Please document that dependency change in git summary.

**Comment by a17r** [comment]
Now you ended up with two commits, please sqash them together!

**Comment by maxsatula** [comment]
Done squashing. Thanks

**Pull request closed without merging**